src/index.ts

```diff
@@ -233,7 +246,8 @@ class LiskMigrator extends Command {
 
 			// Create new DB instance based on the snapshot path
 			cli.action.start('Creating database instance');
-			const db = new KVStore(snapshotPath);
+			const snapshotFilePathExtracted = join(dataDir, 'blockchain.db');
```

Comment:
Same for the extracted snapshot directory.

Reply:
Also defining a constant for the `blockchain.db` filename could be helpful. it can be referred in the test file as well.